### PR TITLE
Add navigation bar and footer

### PR DESCRIPTION
**Pull request by jason17h**
### Notion ticket link

[Frontend: Navbar](https://www.notion.so/uwblueprintexecs/086c1b564dbf478faa649a8880e34b81?v=9d3f8002a0d1477ba395616dfc082af7&p=5e9792fc9722465ca98b7b875d05f650)

### Figma links

Navbar: https://www.figma.com/file/ad7RNFErTGP7HkL52edTwQ/SDC-%2F-External-Application?node-id=1046%3A14703
Language selection: https://www.figma.com/file/ad7RNFErTGP7HkL52edTwQ/SDC-%2F-External-Application?node-id=1046%3A14559
Footer: https://www.figma.com/file/ad7RNFErTGP7HkL52edTwQ/SDC-%2F-External-Application?node-id=1046%3A14609

### Implementation description

- Created navigation bar to be added to the top of all (or most) pages on the UI
- Created footer to be added to the bottom of all (or most) pages on the UI
- Created a new component, `SDCWrapper`, which includes the navbar at the top and footer at the bottom of the component. New components/pages that have the navbar and footer should be wrapped in this component.

NOTE: 
- The header URLs do not redirect anywhere yet except for the "Sign In" button (but the links in the footer do)
- The language selection does not currently do anything

### Steps to test

1. Checkout branch go to http://localhost:3000/
2. Play around with links 

### What should reviewers focus on?

- Appearance rather than functionality for now

### Checklist

-   [x] My PR name is descriptive and in imperative tense
-   [x] I have run the linter
-   [x] I have requested a review from the PL, and/or other devs who have background knowledge on this PR or who will be building on top of this PR

### Screenshots

Signed out:
image

Signed in:
Screen Shot 2021-07-19 at 10 37 16 AM

Language selection:
Screen Shot 2021-07-19 at 10 38 26 AM

Hover over navbar link:
Screen Shot 2021-07-19 at 10 39 01 AM

Hover over sign in button:
image
Screen Shot 2021-07-19 at 10 42 04 AM

Example with content:
image

### CODE REVIEW

File: pages/index.tsx
Comment (feng-eric) [inline pages/index.tsx]: image

Can the footer be moved to the bottom?

File: src/components/Footer.tsx
Comment (feng-eric) [inline src/components/Footer.tsx:52]: When I click on these, there seems to be space after the text. Can this be removed?
image

File: src/components/Navbar.tsx
Comment (feng-eric) [inline src/components/Navbar.tsx]: Perhaps we can set this as a variable to make it more readable.

Comment (feng-eric) [reply src/components/Navbar.tsx]: Also, I think we should download and store the images ourselves. This probably won't happen but in case the image gets removed on their end, it would mess up on our app too.

File: src/definitions/chakra/theme/index.ts
Comment (feng-eric) [inline src/definitions/chakra/theme/index.ts:5]: I believe @mahadzaryab1 added Roboto font. Which one should we use? For the one that's not used, let's remove the dependencies.

**Review by feng-eric — COMMENTED** [review]
Left some comments. It looks really good!

File: src/definitions/chakra/theme/index.ts
Comment (jason17h) [reply src/definitions/chakra/theme/index.ts:5]: The Figma uses Poppins

File: pages/index.tsx
Comment (jason17h) [reply pages/index.tsx]: It technically is at the bottom, there just isn't enough content to fill the page which is why it looks like it's in the middle. I think it would be up to the parent component to just ensure that the minimum height of the container is 100% of the screen size so that the footer always touches the bottom of the screen, even if there isn't a lot of content

File: pages/_app.tsx
Comment (feng-eric) [inline pages/_app.tsx]: Please remove this import if roboto isn't being used. Thanks!

File: package.json
Comment (feng-eric) [inline package.json]: Could you remove this dependency if roboto isn't being used?

**Comment by jason17h** [comment]
Here's how it looks with the lo-fi login page as an example where there's actually content on the screen:

image

File: src/components/Footer.tsx
Comment (feng-eric) [inline src/components/Footer.tsx]: 2021?

File: src/components/Navbar.tsx
Comment (feng-eric) [inline src/components/Navbar.tsx]: Can we store this logo ourselves?

File: src/components/Navbar.tsx
Comment (feng-eric) [inline src/components/Navbar.tsx:23]: Can you leave a comment so we can go back to it in case we want to convert to svg?

**Review by feng-eric — APPROVED** [review]
Had one comment but it LGTM. Let's get one more approval before you merge it in please!

**Review by christinaruu — APPROVED** [review]

**Pull request merged**
